lib/rules/button-icon-a11ytitle.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
     },
     fixable: null,
     messages: {
-      'button-icon-a11ytitle': `Icon-only button requires 'a11yTitle' or 'aria-label' describing the action of the button.`,
+      'button-icon-a11ytitle': `Icon-only button requires 'a11yTitle', 'aria-label', or 'tip' describing the action of the button. Use 'tip' if it would be valuable for the description to appear on hover.`,
```

Comment:
Wondering if we should specify that 'tip' must be a string for it to be used as the aria-label on the button. The current message makes it sound like as long as 'tip' is defined on button this check will pass.

Reply:
Good question. If they implement a custom tip, there is a different flag in this rule that will help catch this. I wonder if that's enough? no strong opinion either way.